_data/cn/stage3.yml

```diff
@@ -0,0 +1,253 @@
+    #!/usr/bin/env node
+    // in the Module Goal
+    export {};
+    console.log(1);
+  presented:
+    - date: November&#xA0;2018
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2018-11/nov-28.md#hash-bang-grammar
+  has_specification: true
+  description: 这个建议是为了配合一些允许使用Shebangs/Hashbang的CLI JS主机的实际使用。这样的主机在传递给目前的JS引擎之前，会先剥离hashbang以生成有效的JS源文本。这将统一和规范这种做法。
+  authors:
+    - Bradley Farias
+  champions:
+    - Bradley Farias
+  tests:
+    - 'https://github.com/tc39/test262/pull/2065'
+- title: 顶层 <code>await</code>
+  id: proposal-top-level-await
+  example: |-
+    // file.html
+    &#x3C;script type=module src=&#x22;a.mjs&#x22;&#x3E;&#x3C;/script&#x3E;
+    // a.mjs
+    await import(&#x22;./b.mjs&#x22;);
+    // b.mjs
+    await import(&#x22;./a.mjs&#x22;);
+  presented:
+    - date: June&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-06/june-6.md#top-level-await-for-stage-3
+  has_specification: true
+  description: 顶层的<code>await</code>让我们依靠模块系统本身来处理所有这些承诺，并确保事情的协调性。
+  authors:
+    - Myles Borins
+  champions:
+    - Myles Borins
+  tests:
+    - 'https://github.com/tc39/test262/pull/2274'
+- title: RegExp Match Indices
+  id: proposal-regexp-match-indices
+  example: |-
+    const re1 = /a+(?&#x3C;Z&#x3E;z)?/;
+    // indices are relative to start of the input string:
+    const s1 = &#x22;xaaaz&#x22;;
+    const m1 = re1.exec(s1);
+    m1.indices[0][0] === 1;
+    m1.indices[0][1] === 5;
+    s1.slice(...m1.indices[0]) === &#x22;aaaz&#x22;;
+    m1.indices[1][0] === 4;
+    m1.indices[1][1] === 5;
+    s1.slice(...m1.indices[1]) === &#x22;z&#x22;;
+    m1.indices.groups[&#x22;Z&#x22;][0] === 4;
+    m1.indices.groups[&#x22;Z&#x22;][1] === 5;
+    s1.slice(...m1.indices.groups[&#x22;Z&#x22;]) === &#x22;z&#x22;;
+    // capture groups that are not matched return &#x60;undefined&#x60;:
+    const m2 = re1.exec(&#x22;xaaay&#x22;);
+    m2.indices[1] === undefined;
+    m2.indices.groups[&#x22;Z&#x22;] === undefined;
+  presented:
+    - date: December&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-3.md#regexp-match-indices-performance-feedback
+  has_specification: false
+  description: ECMAScript RegExp Match Indicies提供了关于捕获的子串相对于输入字符串开始的起始和结束索引的附加信息。
+  authors:
+    - Ron Buckton
+  champions:
+    - Ron Buckton
+  tests:
+    - 'https://github.com/tc39/test262/pull/2309'
+- title: <code>Atomics.waitAsync</code>
+  id: proposal-atomics-wait-async
+  presented:
+    - date: December&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-4.md#atomicswaitasync-for-stage-3
+  has_specification: true
+  description: 为ECMAScript提出一个 "异步原子等待 "的建议，主要用于不允许阻塞的代理。
+  authors:
+    - Lars Hansen
+  champions:
+    - Shu-yu Guo
+    - Lars Hansen
+  tests:
+    - 'https://github.com/tc39/test262/issues/2511'
+- title: <code>.item()</code>
+  id: proposal-item-method
```

Comment:
```suggestion
  id: proposal-at-method
```

Reply:
https://github.com/tc39/proposal-relative-indexing-method

Reply:
I will update this independently, as this will need to be done across languages. 

In fact, we might want to separate the translated fields (title, description) from the rest of the fields. I need to think about that more.